Documentation (#22)
jupyterlab/jupyterlab-builder: README.md

## Why extracting the build tools?

- This would also solve some chicken-and-egg problems like jupyterlab/jupyterlab_pygments#23.
- Isolating the builder functionalities will simplify the work
of core and extension developers who can now focus on their respective parts of the
codebase instead of the earlier intertwined code.

> ## How to install the package?

Reviewer: Let's write some more prose:

```suggestion
## How to install the package?

Execute the following command in a terminal:

```

```
pip install jupyter_builder
```
## What does it do?

> - Provides a CLI for building Jupyter extensions. There are 3 subcommands-

Reviewer: Small typo

```suggestion
- Provides a CLI for building Jupyter extensions. There are 3 subcommands
```

  - `build` : Builds the Jupyter extension JavaScript assets to be consumed by the Jupyter app.
      ```

>       jupyter-builder build (path to extension folder)

Reviewer: For command line optional parameters are usually in square brackets `[]` and required parameters are between `<...>`

```suggestion
      jupyter-builder build <path to extension folder>
```

      ```

>   - `develop` :  Install the Jupyter extension JavaScript assets in dev mode for consumption in the Jupyter app. It similar to [editable install mode of pip](https://setuptools.pypa.io/en/latest/userguide/development_mode.html)

Reviewer: I found a better link for the editable install mode:

```suggestion
  - `develop` :  Install the Jupyter extension JavaScript assets in dev mode for consumption in the Jupyter app. It similar to [editable install mode of pip](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs)
```

      ```
      jupyter-builder develop --overwrite (path to extension folder)
      ```
  - `watch` : Automatically rebuild the development JavaScript assets when one file is changed to ease development.
      ```
      jupyter-builder watch (path to extension folder)
      ```
- Provides a NPM package manager: `jlpm`

> ## How to uninstall the package?

Reviewer: let's write some text
```suggestion
## How to uninstall the package?

Execute the following command in a terminal:

```
